openproblems/tools/normalize.py

```diff
@@ -14,6 +14,9 @@
         """,
     args="sce, min.mean=0.1",
     body="""
+    if(class(sce@assays@data$X)[[1]]!="dgCMatrix"){ # dgRMatrix conversion bug
```

Comment:
What exactly is the bug here? I prefer not to convert from sparse -> dense -> sparse.

Reply:
The bug comes from the conversion from anndata to sce. More specifically, sometimes np.sparse will be converted to dgRMatrix ([sparse but row-oriented matrix as adata.X](https://github.com/theislab/zellkonverter/issues/34)), which is essentially worthless in R.

Found a way to do it without conversion to dense.